packages/SwingSet/package.json

```diff
@@ -53,6 +55,7 @@
     "@babel/core": "^7.5.0",
     "@babel/generator": "^7.6.4",
     "anylogger": "^0.21.0",
+    "ava": "^3.15.0",
```

Comment:
Last time we tried listing `ava` in dependencies rather than devDependencies, bad things happened. Attn @kriskowal

Reply:
it seems to work for me, though I've only run a few tests. Draft PR #2686 is running thru ci now.

https://github.com/Agoric/agoric-sdk/commit/517af5a37ba46ea484e4b9ca12a6542c321eb4f8